internal/metrics/metrics.go

```diff
@@ -134,6 +136,24 @@ func NewMetrics(registry *prometheus.Registry) *Metrics {
 				Help: "Total number of times DAG has been rebuilt since startup",
 			},
 		),
+		DAGRebuildSeconds: prometheus.NewSummary(
+			prometheus.SummaryOpts{
+				Name: DAGRebuildSeconds,
+				Help: "Duration in seconds of DAG rebuilds",
+				Objectives: map[float64]float64{
+					0.1:  0.01,
```

Comment:
thoughts on choice/granularity of quantiles here? i'm not sure how useful some of the in between percentiles might be, wdyt about doing something like 0, .25, .5, .75, .9, .95, .99, 1? (so a little less granularity but still a good set of summary buckets) could omit the 0 and 1 quantiles if we dont care about max/min values

Reply:
No particular reasoning on the chosen values, was just looking to have enough data to understand what's happening -- happy to switch to yours

Reply:
See https://github.com/projectcontour/contour/pull/5009/commits/5389bd0f2a5ca20d529dcc524911fa0e7699e11b